tryton/tryton/gui/window/view_form/view/list.py

```diff
@@ -1210,9 +1210,12 @@ def reset(self):
     def display(self, force=False):
         self.treeview.display_counter += 1
         current_record = self.record
+        model = self.treeview.get_model()
+        if current_record and current_record not in model.group:
+            current_record = None
         if (force
                 or not self.treeview.get_model()
```

Comment:
```suggestion
                or not model
```
On peut réutiliser la variable `model` , ici ? :thinking: